lms/djangoapps/organizations/tests.py

```diff
@@ -103,14 +104,36 @@ def do_get(self, uri):
         response = self.client.get(uri, headers=headers)
         return response
 
-    def do_delete(self, uri):
+    def do_delete(self, uri, data):
         """Submit an HTTP DELETE request"""
         headers = {
             'Content-Type': 'application/json',
             'X-Edx-Api-Key': str(TEST_API_KEY),
         }
-        response = self.client.delete(uri, headers=headers)
```

Comment:
Any particular reason why client.delete() was changed to client.put(REQUEST_METHOD='DELETE') to test the DELETE method of the API call?